loopy/schedule/__init__.py

```diff
@@ -536,8 +537,13 @@ def start(self):
         self.start_time = time()
 
 
-class ScheduleDebugInputError(Exception):
-    pass
+class ScheduleDebugInput(ScheduleDebugInputError):
```

Comment:
Hah. So the `pep8-naming` isn't smart enough to figure out `ScheduleDebugInput` is derived from `Exception` :rofl:.

Reply:
TBH, I really question their judgment on introducing that warning...